pkg/source/openstack/client.go

```diff
@@ -325,6 +317,8 @@ func (c *Client) IsPoweredOff(vm *migration.VirtualMachineImport) (bool, error)
 }
 
 func (c *Client) GenerateVirtualMachine(vm *migration.VirtualMachineImport) (*kubevirt.VirtualMachine, error) {
+	var boolFalse = false
+	var boolTrue = true
```

Comment:
Maybe we can just declare those variables on top of file to reduce repeated declaration?

Reply:
the variables are not needed anywhere other than for pointer references in vm spec. there is no immediate need to increase their scope.